docs/ENVIRONMENT.md

```diff
@@ -260,6 +260,9 @@ All tools:
   it is either set to `system` or `user` depending on whether the NSS/PAM
   module is called by systemd in `--system` or `--user` mode.
 
+* `$SYSTEMD_FORCE_LEGACY_JOB_REMOVED_SIGNAL` — always send the legacy `JobRemoved`
+  signal, even if all clients called `SubscribeWithFlags` instead of `Subscribe`.
```

Comment:
What happens in no one called `SubscribeWithFlags`? Shouldn't this say "always send …, even if nobody subscribed to it"? (I don't know, I'm just starting to read the patch.)